skiko/src/commonTest/kotlin/org/jetbrains/skia/util/assertCloseEnough.kt

```diff
@@ -21,10 +23,39 @@ internal fun assertCloseEnough(expected: Matrix33, actual: Matrix33?, epsilon: F
     expected.mat.zip(actual!!.mat).forEach { (a, b) -> assertCloseEnough(a, b, epsilon) }
 }
 
-
 internal fun assertCloseEnough(expected: Color4f, actual: Color4f?, epsilon: Float = EPSILON) {
     assertCloseEnough(expected.r, actual!!.r, epsilon)
     assertCloseEnough(expected.g, actual.g, epsilon)
     assertCloseEnough(expected.b, actual.b, epsilon)
     assertCloseEnough(expected.a, actual.a, epsilon)
-}
\ No newline at end of file
+}
+
+internal fun assertCloseEnough(expected: Rect, actual: Rect?, epsilon: Float = EPSILON) {
+    assertCloseEnough(expected.left, actual!!.left, epsilon)
```

Comment:
should we have nullable Rect here? after all we can do the !! while passing as param if needed.

Reply:
We have nullable `actual` in other assertions, so let it be just for ergonomics — null and equality checks at once